### PR TITLE
Data Cleaning/Viz- Subplot Method Arguments

### PR DESCRIPTION
**Pull request by rayswebpresence**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/data-cleaning-and-visualizations.english.md [hunk 1]
```diff
@@ -14,11 +14,12 @@ videoId: mHjxzFS5_Z0
 
 ```yml
 question:
-  text: Question
+  text: 'When using MatPlotLib's global API, the following code does what in what order?:
```
Comment (erictleung) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/data-cleaning-and-visualizations.english.md:17]: Matplotlib is just capitalized, the apostrophe here in `Matplolib's` needs to be escaped, and the original wording of the question felt awkward. Here's a suggested change

```suggestion
  text: 'When using Matplotlib''s global API, what does the order of numbers mean here?:
```

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/data-cleaning-and-visualizations.english.md [hunk 2]
```diff
@@ -14,11 +14,12 @@ videoId: mHjxzFS5_Z0
...
+  <pre>plt.subplot(1, 2, 1)</pre>
   answers:
-    - one
-    - two
-    - three
+    - 'My figure will have one column, two rows, and I am goin to start drawing in the first (left) plot.'
```
Comment (erictleung) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/data-cleaning-and-visualizations.english.md:20]: Typo

```suggestion
    - 'My figure will have one column, two rows, and I am going to start drawing in the first (left) plot.'
```

**Review by erictleung — APPROVED** [review]
See suggestions

**Review by erictleung — CHANGES_REQUESTED** [review]
See comments

**Review by beaucarnes — APPROVED** [review]

**Pull request merged**